dat/missions/dvaered/colonel-transport.lua

```diff
@@ -0,0 +1,164 @@
+--[[
+<?xml version='1.0' encoding='utf8'?>
+<mission name="Dvaered Colonel Escort">
+ <unique />
+ <priority>4</priority>
+ <chance>43</chance>
+ <location>Bar</location>
+ <faction>Dvaered</faction>
+ <cond>require("misn_test").reweight_active()</cond>
+</mission>
+--]]
+--[[
+
+   Mission: Escort a Dvaered colonel
+   Description: Small mission where you escort a Dvaered Arsenal.
+                This is a one-off that's not part of any major
+                storyline; the in-game purpose will remain a mystery.
+
+--]]
+
+local escort = require "escort"
+local fmt = require "format"
+local lmisn = require "lmisn"
+local neu = require "neutral"
+local vn = require "vn"
+local vni = require "vnimage"
+
+local reward = 750e3
+
+local source_system = nil
+
+local npc_name = _("Dvaered Colonel")
+local npc_portrait = nil
+local npc_image = nil
+
+local m = vn.newCharacter( npc_name, {image=npc_image} )
```

Comment:
You can't just define the character here, because they get added to each scene. You have to define them after a `vn.scene()`.